contracts/solidity/NFTXVaultUpgradeable.sol

```diff
@@ -73,7 +73,6 @@ contract NFTXVaultUpgradeable is
         bool _is1155,
         bool _allowAllItems
     ) public virtual override initializer {
-        __Ownable_init();
```

Comment:
Is this not required to assign the owner as the `msg.sender` or as another EOA? There are a couple of `onlyOwnerIfPaused` calls that I believe would now have `address(0)` as the `owner()`?

Reply:
Instead of each vault having the ownable functionality, the owner function has been overwritten to always return vaultFactory.owner()

so when we change the owner of the vaultFactory, then all vaults automatically reference that.

Initializing Owable here would have just consumed gas.